packages/ember-glimmer/lib/environment.js

```diff
@@ -132,7 +132,7 @@ export default class Environment extends GlimmerEnvironment {
     super(...arguments);
     this.owner = owner;
 
-    this.uselessAnchor = document.createElement('a');
+    this.uselessAnchor = this.getAppendOperations().createElement('a');
```

Comment:
This is only used for `protocolForURL`, which needs to be overridden in Node anyway